pkg/platform/local/client/store.go

```diff
@@ -310,7 +310,14 @@ func (s *Store) serializeAndWriteFileContents(resourcePath string, resourceConfi
 }
 
 func (s *Store) getResourcePath(resourceDir string, resourceNamespace string, resourceName string) string {
```

Comment:
This function is used for Functions, Projects and functions events, for all crud operations.
Will the change somehow affect flows other than delete?

Reply:
@TomerShor no, it won't, because we will go in this condition only if there is a space in resourceName, which is impossible to happen in the recent versions